pages/osx/bless.md

```diff
@@ -1,14 +1,15 @@
 # bless
 
 > Set volume bootability and startup disk options.
+> More information: <https://ss64.com/osx/bless.html>.
 
 - FOLDER MODE: bless a volume with only Mac OS X or Darwin, and create the BootX and boot.efi files as needed:
 
-`bless --folder {{/Volumes/Mac OS X/System/Library/CoreServices}} --bootinfo --bootefi`
+`bless --folder "{{/Volumes/Mac OS X/System/Library/CoreServices}}" --bootinfo --bootefi`
 
 - MOUNT MODE: set a volume containing either Mac OS 9 and Mac OS X to be the active volume:
```

Comment:
Oh, I see what you mean here @bl-ue. Yeah, it would perhaps be worth changing these while we're at it.
```suggestion
- Mount mode: set a volume containing either Mac OS 9 and Mac OS X to be the active volume:
```

Reply:
Why not just get rid of them entirely?

Reply:
I mean you could, but I don't have a mac to test these on and I'm unsure as to whether they have semantic meaning or not.

Reply:
They don't even need to be tested.  The 'modes' were taken directly from the first long option in the example, e.g. `MOUNT MODE` because of `--mount`, `NETBOOT MODE` because of `--netboot`, etc.

I vote to get rid of them.

Reply:
Hrm, ok then